di-ipv-credential-issuer-stub/src/main/resources/data/criStubEvidencePayloads.json

```diff
@@ -33,6 +33,19 @@
         "strengthScore": 4
       }
     },
+    {
+      "criType": "Fraud Check (Stub)",
+      "label": "Failed fraud check (M1C)",
```

Comment:
Could we label this intl' address - as there will be a different VC for M1C for Experian being down